### ISSUE TITLE
Version compiled for 2.12 

### ISSUE THREAD

**Issue by hmf** [deeplearning4j/nd4s#96]
Hello,

I am attempting to use ND4S with Scala version 2.12.  
My first attempt failed because I have not found the binaries for 2.12.
I then saw issue #82 and proceeded to clone, compile and test ND4S. 
This however failed with the following unresolved dependencies:

```
org.nd4j#nd4j-api;0.6.1-SNAPSHOT: not found
org.nd4j#nd4j-native-platform;0.6.1-SNAPSHOT: not found
```
Is their any way I can generate the binary for 2.12 and 
the latest ND4J repository libraries (0.6.0)?

Just in case I am wrong,  could anyone confirm that 
no ND4S binaries for 2.12 exists in the repositories?

TIA.

**Comment by saudet** [comment]
Try to make your modifications on the source code here:
https://github.com/deeplearning4j/nd4s/tree/nd4s-0.6.0
AFAIU, it should work just fine. You could then send a pull request! Thanks

**Comment by agibsonccc** [comment]
@hmf we will be doing a release tomorrow. We can do a brief update on nd4s for scala 2.12 then.

**Comment by hmf** [comment]
@agibsonccc I would appreciate it. I tried to do this with the  0.6.0 branch. I change my local repo to the 0.6.0 branch and set the Scala version to 2.12. However this still failed. I now have the following missing dependencies. 

```
com.nativelibs4java#scalaxy-loops_2.12;0.3.4: not found
org.scalatest#scalatest_2.12;2.2.6: not found
org.scalacheck#scalacheck_2.12;1.12.5: not found
org.scalanlp#breeze_2.12;0.12: not found
com.github.julien-truffaut#monocle-core_2.12;1.2.0: not found
```
I will wait for your update. 
Thank you.

**Comment by hmf** [comment]
Hello,

I have tried to use Scala version 2.12.1 and failed again. According to the Maven repo the latest version should be 0.7.2. However, it no 2.12.x version seems to be available. I get:

```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: org.nd4j#nd4s_2.12;0.7.0: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 
[warn] 	Note: Unresolved dependencies path:
[warn] 		org.nd4j:nd4s_2.12:0.7.0 (/home/hmf/git/dl4jtest/build.sbt#L68-69)
[warn] 		  +- default:dl4jtest_2.12:0.1
```

I have tried several versions to no avail. Any way I can get around this?

TIA

**Comment by saudet** [comment]
@hmf Please try to build from source. When all dependencies of nd4s also become available for 2.12, then we will be able to release nd4s for 2.12 as well.

**Comment by agibsonccc** [comment]
https://github.com/deeplearning4j/nd4s/pull/103
